revprxy.yaml

```diff
@@ -0,0 +1,160 @@
+- hosts: 34.121.46.70
+  gather_facts: no
+  remote_user: ubuntu
```

Comment:
any specific reason we are going for ubuntu user?

Reply:
sure, it does not look appropriate having this @ ubuntu.....creating a specific user named, "nginx"  and running the playbook there instead.